jta-crash-rec/.github/workflows/quickstart_jta-crash-rec_ci.yml

```diff
@@ -0,0 +1,15 @@
+name: WildFly jta-crash-rec Quickstart CI
```

Comment:
@jmfinelli This is in the wrong folder

Reply:
Thanks @kabir. I moved it to the right folder (I hope)